File: python_projects/src/strings/lengthOfLastWord.py
class Solution:
    # @param A : string
    # @return an integer
    def lengthOfLastWord(self, A):
        words = A.split()
        if words == []:
            return 0
        return len(words[-1])

a = Solution()
inp = '  '
str = a.lengthOfLastWord(inp)
print ('Length of Last Word = ', str)

File: LeetCode_java/src/strings/LengthOfLastWord.java
/*
 * Given a string s consists of upper/lower-case alphabets and empty space characters ' ', return the length of last word in the string.
 * If the last word does not exist, return 0.
 * 
Input: "Hello World"
Output: 5
 */

package strings;

public class LengthOfLastWord {
	public int lengthOfLastWord(String s) {
		// Remove all beginning/trailing spaces
		s = s.trim();
		if (s==null || s.length()==0)
			return 0;
		String[] words = s.split(" ");
		return words[words.length-1].length();
    }
}
